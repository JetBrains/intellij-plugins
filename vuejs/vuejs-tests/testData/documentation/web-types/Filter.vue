<template>
  <div :title="12 | my<caret>Filter"></div>
</template>
