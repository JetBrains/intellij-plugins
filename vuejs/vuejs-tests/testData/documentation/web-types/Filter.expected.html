<div class='definition'><pre>myFilter</pre></div><div class='content'><p>Filter <i>description</i>.</p></div><table class='sections'><tr><td valign='top' class='section'><p>Library:</td><td valign='top'><p>test-lib</td></table>